lighthouse-cli/cli-flags.js

```diff
@@ -53,11 +53,17 @@ function getFlags(manualArgv) {
           'lighthouse <url> --quiet --chrome-flags="--headless"',
           'Launch Headless Chrome, turn off logging')
       .example(
-          'lighthouse <url> --extra-headers "{\\"Cookie\\":\\"monster=blue\\", \\"x-men\\":\\"wolverine\\"}"',
+          'lighthouse <url> --extra-headers "{\\"x-men\\":\\"wolverine\\"}"',
           'Stringify\'d JSON HTTP Header key/value pairs to send in requests')
       .example(
           'lighthouse <url> --extra-headers=./path/to/file.json',
           'Path to JSON file of HTTP Header key/value pairs to send in requests')
+      .example(
+          'lighthouse <url> --extra-cookies "[{\\"name\\":\\"session_id\\",\\"value\\":\\"x-men\\" }]"',
```

Comment:
what motivated an array instead of matching the headers with something like `{"session_id": "x-men"}`, just to match the protocol format?

I guess if we want to support other cookie properties like domain and expiration this makes sense but it seems a tad verbose for the common case and the lack of parallelism with headers is unfortunate.

Reply:
@patrickhulce please take a look at the following https://github.com/GoogleChrome/lighthouse/pull/9170#issuecomment-500606424